Slack.Automation/Promact.Core.Repository/TaskMailRepository/TaskMailRepository.cs

```diff
@@ -344,7 +367,7 @@ public TaskMailRepository(IRepository<TaskMail> taskMail, IStringConstantReposit
                 catch (SmtpException ex)
                 {
                     // error message will be send to email. But leave will be applied
-                    questionText = string.Format("{0}. {1}", _stringConstant.ErrorOfEmailServiceFailureTaskMail, ex.Message.ToString());
+                    questionText = string.Format(_stringConstant.ReplyTextForSMTPExceptionErrorMessage, _stringConstant.ErrorOfEmailServiceFailureTaskMail, ex.Message.ToString());
```

Comment:
After catching exception. you are not writing log or throwing it. How will the development team know there was exception on server?